### PR TITLE
View segments in islandora

### PR DESCRIPTION
**Pull request by dynac01**
Segments and newspapers are viewable in the image viewer.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -442,4 +442,7 @@ composer.lock
 
 #### Vagrant
 .vagrant/
-*.box
\ No newline at end of file
+*.box
+
+#### Temp
+Vagrantfile
```
Comment (Alexander-Cairns) [inline .gitignore]:
```suggestion
```
Vagrantfile should be tracked

**Review by Alexander-Cairns — CHANGES_REQUESTED** [review]
Looks great! Just remove the Vagrantfile from the gitignore.

**Comment by dynac01** [comment]
> Looks great! Just remove the Vagrantfile from the gitignore.

Done

**Review by Alexander-Cairns — APPROVED** [review]
Looks great will merge now

**Pull request merged**